hat/assets/js/apps/Iaso/domains/orgUnits/hooks.js

```diff
@@ -27,16 +27,14 @@ export const useOrgUnitDetailData = (
                 onSuccess: ou => setCurrentOrgUnit(ou),
             },
         );
-    const groupsUrl = useMemo(() => {
-        const baseUrl = '/api/groups/';
-        if (isNewOrgunit) {
-            return `${baseUrl}?&defaultVersion=true`;
-        }
-        if (originalOrgUnit?.source_id) {
-            return `${baseUrl}?&dataSource=${originalOrgUnit.source_id}`;
-        }
-        return baseUrl;
-    }, [isNewOrgunit, originalOrgUnit?.source_id]);
+    const groupsDataSourceQueryParams = originalOrgUnit?.source_id
+        ? `?dataSource=${originalOrgUnit.source_id}`
+        : undefined;
+    const groupsQueryParams = isNewOrgunit
+        ? '?defaultVersion=true'
+        : groupsDataSourceQueryParams;
+    const groupsApiUrl = '/api/groups/dropdown/';
+
```

Comment:
It's a bit of gymastic but:
- strings don't need to be memoized
- it allows me to pass just the params to the queryKey,  so all group dropdown hooks have the same queryKey structure